install-prod.sh

```diff
@@ -44,6 +44,17 @@ check_dependencies() {
 	fi
 }
 
+configure_sed(){
```

Comment:
I think we can simplify this with a one line definition at the beginning of the script along with other globals so that it's explicit.

`SED_INPLACE=$(if [[ "$(uname)" == "Darwin" ]]; then echo "-i ''"; else echo "-i"; fi)`